File: myListViewItem.vb
Public Class myListViewItem
    Inherits ListViewItem

    ' Public FilePath As String
    Public strName As String
    Public strType As String
    Public intID As Integer
    Public boolActive As Boolean

    Sub New(ByVal Name As String)
        MyBase.New(Name)
    End Sub

    Sub New(ByVal Name As String, ByVal Type As String, ByVal ID As Integer, ByVal imageIndex As Integer)

        MyBase.New(Name, imageIndex)
        MyBase.Tag = Type
        intID = ID

    End Sub

    Sub New(ByVal Name As String, ByVal Type As String, ByVal ID As Integer)
        MyBase.New()
        strName = Name
        MyBase.Tag = Type
        intID = ID
        Me.Text = Name

    End Sub

End Class

File: CS/myListViewItem.cs
using System.Drawing;
using System.Data;
using System.Windows.Forms;
using System.Collections;
using Microsoft.VisualBasic;
using System.Diagnostics;
using System;

namespace bftm
{
	public class myListViewItem : ListViewItem
	{
		
		
		// Public FilePath As String
		public string strName;
		public string strType;
		public int intID;
		public bool boolActive;
		
		public myListViewItem(string Name) : base(Name)
		{
		}
		
		public myListViewItem(string Name, string Type, int ID, int imageIndex) : base(Name, imageIndex)
		{
			
			base.Tag = Type;
			intID = ID;
			
		}
		
		public myListViewItem(string Name, string Type, int ID)
		{
			strName = Name;
			base.Tag = Type;
			intID = ID;
			this.Text = Name;
			
		}
		
	}
	
}
